src/wagtail_ai/text_splitters/length.py

```diff
@@ -0,0 +1,55 @@
+import decimal
+import logging
+import math
+import re
+
+from .base import BaseTextSplitterLength
+
+logger = logging.getLogger(__name__)
+
+
+class NaiveTextSplitterLength(BaseTextSplitterLength):
+    """
+    Text splitter length function that estimates how many tokens there
+    are in the text rather than actually giving the correct reading.
+
+    The goal is that we have a safe function that can be used accross
+    all the popular models.
+
+
+    Using OpenAI as a guide:
+    - https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them
+
+    - 4 characters is the average length of one token.
+    - 3/4 words is the average length of one token.
+    """
+
+    characters_per_token: int | float | decimal.Decimal = 4
+    words_per_token: int | float | decimal.Decimal = 0.75
+    final_multiplier: int | float | decimal.Decimal = 1
+
+    def get_splitter_length(self, text: str) -> int:
```

Comment:
**question:** Should this implement a buffer for safety in cases where we under-estimate? Is that the intention of `final_multiplier`?